advanced/editor-command-identifiers.md

```diff
@@ -75,6 +75,7 @@ The commands on the following table are provided by the {{site.productname}} edi
 | mceSetContent        | Sets the contents of the editor. The value is the contents to set as the editor contents.                                                                                                                                                                                                   |
 | mceToggleFormat      | Toggles a specified format by name. The value is the name of the format to toggle. For a list of options, see: [Content formatting options - Built-in formats]({{ site.baseurl }}/configure/content-formatting/#built-informats).                                                                               |
 | ToggleSidebar        | Closes the currrent sidebar, or toogles the sidebar if the sidebar name is provided as a value (_`<sidebar-name>`_).                                                                                                                                                                        |
+| ToggleToolbarDrawer  | Toggles the toolbar drawer. For information on toolbars, see: [User interface options - Toolbar]({{site.baseurl}}/configure/editor-appearance/#toolbar).                                                                                                                                                                        |
```

Comment:
Does this need the `{{ site.requires_5_5v }}` note, to indicate it only works in 5.5 or above?

Reply:
Fixed